internal/pkg/template/templates/workloads/services/lb-web/cf.yml

```diff
@@ -174,6 +174,9 @@ Resources:
     Type: AWS::ElasticLoadBalancingV2::TargetGroup
     Properties:
       HealthCheckPath: {{.HTTPHealthCheck.HealthCheckPath}} # Default is '/'.
+{{- if .HTTPHealthCheck.Port}}
+      HealthCheckPort: "{{.HTTPHealthCheck.Port}}" # Default is 'traffic-port'.
```

Comment:
nit: 
```suggestion
      HealthCheckPort: {{.HTTPHealthCheck.Port}} # Default is 'traffic-port'.
```

Reply:
Thanks, @efekarakus, for the suggestion. I wan't initially sure if CloudFormation would parse a number as a string, since the `HealthCheckPort` type is `string` according to the CF docs. I did a test with just passing `HealthCheckPort: 3000` and it worked though.